### PR TITLE
[zh] Sync Kubernetes Native Sidecar in Istio blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Translate Kubernetes Native Sidecar in Istio blog into Chinese:
```
content/zh/blog/2023/native-sidecars/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2023/native-sidecars/index.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+## 从根源解决 {#fixing-the-root-cause}
+
+虽然 Istio 中日益复杂的变通方案可以帮助减轻 Istio 用户的痛苦，
+但理想情况下，所有相关情况都应该正常运转 - 而不仅仅是针对 Istio。
+幸运的是，Kubernetes 社区一直努力直接在 Kubernetes 中解决这些问题。
+
+在 Kubernetes 1.28 中，合并了一项为 Sidecar 添加原生支持的新功能，
+这为 5 年多的持续工作画上了句号。通过该功能，我们的所有问题都可以得到解决，而无需其他变通方案！
+
+当我们进入“GitHub Issue 名人堂”时，
```
Comment (my-git9) [inline content/zh/blog/2023/native-sidecars/index.md:51]:
```suggestion
当我们进入 “GitHub Issue 名人堂”时，
```

Comment (wilsonwu) [reply content/zh/blog/2023/native-sidecars/index.md:51]: I think the full-width double quotation mark connect Chinese and English characters does not need space, is that right?

File: content/zh/blog/2023/native-sidecars/index.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
+---
+title: Istio 中的 Kubernetes Native Sidecar
```
Comment (windsonsea) [inline content/zh/blog/2023/native-sidecars/index.md:2]:
```suggestion
title: 在 Istio 中使用 Kubernetes 原生 Sidecar
```

File: content/zh/blog/2023/native-sidecars/index.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+description: 使用 Istio 演示新的 SidecarContainers 功能。
+publishdate: 2023-08-15
+attribution: "John Howard (Google); Translated by Wilson Wu (DaoCloud)"
+keywords: [istio,sidecars,kubernetes]
+---
+
+如果您曾经听说过有关服务网格的些许信息，
+那应该就是它们使用 Sidecar 模式进行工作：代理服务器与您的应用程序代码部署在一起。
```
Comment (windsonsea) [inline content/zh/blog/2023/native-sidecars/index.md:10]:
```suggestion
那就会知道它是以 Sidecar 模式工作的：与应用代码并列部署一个代理服务器。
```

File: content/zh/blog/2023/native-sidecars/index.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+Sidecar 就是这样：它是一种模式。到目前为止，在 Kubernetes 中还没有对 Sidecar 容器进行正式支持。
```
Comment (windsonsea) [inline content/zh/blog/2023/native-sidecars/index.md:11]:
```suggestion
Sidecar 就是这样：它是一种模式。在今日（8 月 15 日，即 Kubernetes 1.28 正式发版之日）之前，Kubernetes 对 Sidecar 容器没有任何形式的正式支持。
```

**Review by windsonsea — COMMENTED** [review]
刚看了个开头，貌似还需要一些时间 review

File: content/zh/blog/2023/native-sidecars/index.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+幸运的是，Kubernetes 社区一直努力直接在 Kubernetes 中解决这些问题。
+
+在 Kubernetes 1.28 中，合并了一项为 Sidecar 添加原生支持的新功能，
+这为 5 年多的持续工作画上了句号。通过该功能，我们的所有问题都可以得到解决，而无需其他变通方案！
+
+当我们进入“GitHub Issue 名人堂”时，
+[这些](https://github.com/kubernetes/kubernetes/issues/25908)在所有 Kubernetes Issue
+中的 #1 和 #6 的两个 [Issue](https://github.com/kubernetes/kubernetes/issues/65502) - 终于被关闭了！
+
+特别感谢参与冲刺的众多个人贡献者。
```
Comment (windsonsea) [inline content/zh/blog/2023/native-sidecars/index.md:55]:
```suggestion
特别感谢让这个功能冲过终点线的众多贡献者。
```

File: content/zh/blog/2023/native-sidecars/index.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+令人惊喜的是！`istio-proxy` 现在以一个 `initContainer` 方式存在。
+
+更具体地说，它是一个被设置了 `restartPolicy: Always` 的 `initContainer`
+（一个新字段，随着 `SidecarContainers` 功能被启用）。
+该设置告诉 Kubernetes 将其视为 Sidecar。
+
+这意味着在代理容器准备好之前，`initContainers` 以及 `containers` 中的所有后续容器将不会被启动。
+此外，即使代理容器仍在运行，Pod 也可以被终止。
+
+### 初始化容器流量 {#init-container-traffic}
```
Comment (windsonsea) [inline content/zh/blog/2023/native-sidecars/index.md:123]:
```suggestion
### Init 容器流量 {#init-container-traffic}
```
[Init 容器](https://kubernetes.io/zh-cn/docs/concepts/workloads/pods/init-containers/)需要 follow k8s 吗

File: content/zh/blog/2023/native-sidecars/index.md [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+
+## Ambient 模式又当如何？ {#what-about-ambient-mode}
+
+去年，Istio 宣布了 [Ambient 模式](/zh/blog/2022/introducing-ambient-mesh/) - Istio
+的一种新数据平面模式，不依赖 Sidecar 容器。那么随着 Ambient 模式的到来，这一切还重要吗？
+
+我会大声地告诉大家“重要”！
+
+虽然当 Ambient 模式用于工作负载时，Sidecar 的影响会减轻，
+但我预计几乎所有大规模 Kubernetes 用户在其 Deployment 中都会存在某种 Sidecar。
```
Comment (windsonsea) [inline content/zh/blog/2023/native-sidecars/index.md:209]:
```suggestion
但我预计几乎所有大规模 Kubernetes 集群的用户在其部署中都会存在某种 Sidecar。
```

File: content/zh/blog/2023/native-sidecars/index.md [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+因此，虽然这种情况可能较少，但对于使用 Sidecar 的情况来说，这仍然是一个巨大的改进。
+
+您可能想了解的相反情况 - 如果我们所有的 Sidecar 问题都得到解决，
+为什么我们还需要 Ambient 模式？解决了这些边车限制后，Ambient 仍然带来了多种好处。
+例如，[这篇博文](/zh/blog/2023/waypoint-proxy-made-simple/)详细介绍了将代理与工作负载分离的必要性。
+
+## 自己尝试一下 {#try-it-out-yourself}
+
+我们鼓励有冒险精神的读者在测试环境中亲自尝试！对这些实验性和 Alpha
+功能的反馈对于确保它们稳定并在推广之前满足预期至关重要。 如果您进行尝试，
```
Comment (windsonsea) [inline content/zh/blog/2023/native-sidecars/index.md:220]:
```suggestion
功能的反馈对于确保它们稳定并在推广之前满足预期至关重要。如果您做了尝试，
```

File: content/zh/blog/2023/native-sidecars/index.md (hunk 6 above)
Comment (wilsonwu) [reply content/zh/blog/2023/native-sidecars/index.md:123]: Yes, I think so.

**Review by my-git9 — APPROVED** [review]
LGTM

**Review by windsonsea — APPROVED** [review]
/lgtm

**Pull request merged**